index.js

```diff
@@ -3,6 +3,7 @@
 module.exports = {
   rules: {
     'block-spacing': [2, 'always'],
+    'brace-style': [2, "1tbs"],
```

Comment:
@dgrabla @dancingrobot84 This setting is actually breaking the coding style of this package 😏 . It requires using single quotes...